components/ws-manager/BUILD.yaml

```diff
@@ -63,3 +64,8 @@ packages:
         # - ["sh", "-c", "./components-ws-manager--app/ws-manager --kubeconfig kubeconfig integration-test objs chart--helm/gitpod | kubectl --kubeconfig kubeconfig apply -f -"]
         # - ["components-ws-manager--app/_tests/manager.test", "-integration-test", "kubeconfig", "-test.run", "TestIntegration", "-test.v"]
         # - ["kubectl", "--kubeconfig", "kubeconfig", "delete", "namespace", "wsman-integration-${__pkg_version}"]
+  - name: kubebuilder-binaries
```

Comment:
@csweichel how I can disable cache for this task? Or better yet, how I can add these binaries to the image build image
(I tried updating the `dev-environment` image without success. Maybe this needs to be done in a different PR?)

Reply:
You can add them to the dev-environment [Dockerfile](https://github.com/gitpod-io/gitpod/blob/master/dev/image/Dockerfile), let werft build the image for you (see werft result page) and then use that image in the `.gitpod.yml` and `.werft/build.yaml`.

Reply:
After going through the code I was left wondering what we'd be using `kubebuilder` for. Was it used for the metrics that were previously part of this PR?

Reply:
Is for [envtest](https://book.kubebuilder.io/reference/envtest.html) The binaries are related to kube-apiserver and etcd